<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FtpUpload</class>
 <widget class="QDialog" name="FtpUpload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>381</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QTreeWidget" name="fileList">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblServer">
       <property name="text">
        <string>Server URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="uploadUrlLineEdit">
       <property name="placeholderText">
        <string>upload url</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPathl">
       <property name="text">
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtPath"/>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="cmdParent">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../tosec.qrc">
         <normaloff>:/images/ftp/images/farious/dir/up.ico</normaloff>:/images/ftp/images/farious/dir/up.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdRemove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../tosec.qrc">
         <normaloff>:/images/ftp/images/farious/dir/delete.ico</normaloff>:/images/ftp/images/farious/dir/delete.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdMkdir">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../tosec.qrc">
         <normaloff>:/images/ftp/images/farious/dir/mkdir.ico</normaloff>:/images/ftp/images/farious/dir/mkdir.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdUpload">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../tosec.qrc">
         <normaloff>:/images/ftp/images/farious/dir/upload.ico</normaloff>:/images/ftp/images/farious/dir/upload.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdDownload">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../tosec.qrc">
         <normaloff>:/images/ftp/images/farious/dir/download.ico</normaloff>:/images/ftp/images/farious/dir/download.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QwwLed" name="widLed" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="fileNameLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>file path</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblUser">
       <property name="text">
        <string>Benutzer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtUser"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Passwort:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtPasswort">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblFtpServer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;FTP Verbindungen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdConnect">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../tosec.qrc">
         <normaloff>:/icon/farious/blue/images/farious/blue/verify.ico</normaloff>:/icon/farious/blue/images/farious/blue/verify.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../tosec.qrc">
         <normaloff>:/icon/farious/blue/images/farious/blue/save.ico</normaloff>:/icon/farious/blue/images/farious/blue/save.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../tosec.qrc">
         <normaloff>:/icon/farious/blue/images/farious/blue/delete.ico</normaloff>:/icon/farious/blue/images/farious/blue/delete.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="selectFileButton">
     <property name="text">
      <string>&amp;Datei ausw√§hlen</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>&amp;Beenden</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwwLed</class>
   <extends>QWidget</extends>
   <header>3dparty/wwWidgets/widgets/qwwled/qwwled.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../tosec.qrc"/>
 </resources>
 <connections/>
</ui>
